stdlib/http/src/main/ballerina/src/http/http_commons.bal

```diff
@@ -245,7 +245,7 @@ function buildResponse(ResponseMessage message) returns Response {
     return response;
 }
 
-# The HEAD remote function implementation of the Circuit Breaker. This wraps the `head()` function of the underlying
+# The HEAD remote function implementation of the Circuit Breaker. This wraps the `head` function of the underlying
```

Comment:
```suggestion
# The HEAD remote function implementation of the Circuit Breaker. This wraps the `head()` function of the underlying
```

Shall we revert this change? This description is misleading. @arukshani Could you please verify this?